script/misc-helper.ts

```diff
@@ -428,13 +428,13 @@ async function getCIHelper(): Promise<CIHelper> {
         for (const arg of commander.args.slice(1)) {
             onlyPRs.add(parseInt(arg, 10));
         }
-        await ci.handleNewMails(mailArchiveGitDir!,
+        await ci.handleNewMails(mailArchiveGitDir,
                                 onlyPRs.size ? onlyPRs : undefined);
     } else {
         process.stderr.write(`${command}: unhandled sub-command\n`);
         process.exit(1);
     }
-})().catch((reason) => {
+})().catch((reason: Error) => {
```

Comment:
I was getting a possible TS error specifying this.  If it is okay, is there a possible use in ci-helper.ts line ~687?

Reply:
> I was getting a possible TS error specifying this.

Oh? I did not...

> If it is okay, is there a possible use in ci-helper.ts line ~687?

I don't quite understand... Are you referring to this? https://github.com/gitgitgadget/gitgitgadget/blob/db20df4bc43489b25ffc6e5355ca7db0d7a49879/lib/ci-helper.ts#L687

If so, no, that cannot be changed. The difference is that the former `catch()` is a _method_ (of the `Promise` class) while latter `catch` is an actual part of a `try { ... } catch(e) { ... }` construct, and that latter `catch` is not allowed to use any data type for the parameter other than `any`.

Sad, for sure.

Reply:
Sorry, I missed that differentiation.